drivers/sensor/nordic/qdec_nrfx/qdec_nrfx.c

```diff
@@ -45,7 +47,7 @@ static void accumulate(struct qdec_nrfx_data *data, int32_t acc)
 	unsigned int key = irq_lock();
 
 	bool overflow = ((acc > 0) && (ACC_MAX - acc < data->acc)) ||
```

Comment:
It would be good to also report this overflow to the user, like the hardware one.